build/artifacts.props

```diff
@@ -71,8 +71,8 @@
     <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.Abstractions" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
     <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.Elm" Category="noship" />
     <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore" Category="ship" AppMetapackage="true" AllMetapackage="true"/>
-    <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Category="noship" Metapackage="false" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.Identity.Service" Category="ship" AppMetapackage="true" AllMetapackage="true" />
+    <PackageArtifact Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Category="noship" />
```

Comment:
@anurse are HealthCheck packages shipping in preview2?

We may need to revert this if so.

Reply:
@anurse confirmed that HealthCheck packages are not shipping in 2.1 at all, not just preview 1

Reply:
@glennc can confirm, but that is my understanding. Certainly they aren't coming in preview1